exercises/practice/rational-numbers/test/rational_numbers_test.exs

```diff
@@ -4,239 +4,157 @@ defmodule RationalNumbersTest do
     end
   end
 
   describe "Absolute value" do
     @tag :pending
     test "Absolute value of a positive rational number" do
-      r = {1, 2}
-      result = {1, 2}
-      assert RationalNumbers.abs(r) == result
+      assert RationalNumbers.abs({1, 2}) == {1, 2}
     end
 
     @tag :pending
     test "Absolute value of a positive rational number with negative numerator and denominator" do
-      r = {-1, -2}
-      result = {1, 2}
-      assert RationalNumbers.abs(r) == result
+      assert RationalNumbers.abs({-1, -2}) == {1, 2}
     end
 
     @tag :pending
     test "Absolute value of a negative rational number" do
-      r = {-1, 2}
-      result = {1, 2}
-      assert RationalNumbers.abs(r) == result
+      assert RationalNumbers.abs({-1, 2}) == {1, 2}
     end
 
     @tag :pending
     test "Absolute value of a negative rational number with negative denominator" do
-      r = {1, -2}
-      result = {1, 2}
-      assert RationalNumbers.abs(r) == result
+      assert RationalNumbers.abs({1, -2}) == {1, 2}
     end
 
     @tag :pending
     test "Absolute value of zero" do
-      r = {0, 1}
-      result = {0, 1}
-      assert RationalNumbers.abs(r) == result
+      assert RationalNumbers.abs({0, 1}) == {0, 1}
     end
   end
 
   describe "Exponentiation of a rational number" do
     @tag :pending
     test "Raise a positive rational number to a positive integer power" do
-      r = {1, 2}
-      n = 3
-      result = {1, 8}
-      assert RationalNumbers.pow_rational(r, n) == result
+      assert RationalNumbers.pow_rational({1, 2}, 3) == {1, 8}
     end
 
     @tag :pending
     test "Raise a negative rational number to a positive integer power" do
-      r = {-1, 2}
-      n = 3
-      result = {-1, 8}
-      assert RationalNumbers.pow_rational(r, n) == result
+      assert RationalNumbers.pow_rational({-1, 2}, 3) == {-1, 8}
     end
 
     @tag :pending
     test "Raise zero to an integer power" do
-      r = {0, 1}
-      n = 5
-      result = {0, 1}
-      assert RationalNumbers.pow_rational(r, n) == result
+      assert RationalNumbers.pow_rational({0, 1}, 5) == {0, 1}
     end
 
     @tag :pending
     test "Raise one to an integer power" do
-      r = {1, 1}
-      n = 4
-      result = {1, 1}
-      assert RationalNumbers.pow_rational(r, n) == result
+      assert RationalNumbers.pow_rational({1, 1}, 4) == {1, 1}
     end
 
     @tag :pending
     test "Raise a positive rational number to the power of zero" do
-      r = {1, 2}
-      n = 0
-      result = {1, 1}
-      assert RationalNumbers.pow_rational(r, n) == result
+      assert RationalNumbers.pow_rational({1, 2}, 0) == {1, 1}
     end
 
     @tag :pending
     test "Raise a negative rational number to the power of zero" do
-      r = {-1, 2}
-      n = 0
-      result = {1, 1}
-      assert RationalNumbers.pow_rational(r, n) == result
+      assert RationalNumbers.pow_rational({-1, 2}, 0) == {1, 1}
     end
   end
 
```

Comment:
Reviewers: note I left the "exponentiation of a real number to a rational number" (not shown in diff so you'll need to expand) intact since changing them would've made the lines long.
Let me know if you think I should change them as well.